src/backend/tests/models/test_record.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+from expungeservice.models.case import Case
+from tests.models.record_object import Record
+from tests.factories.case_factory import CaseFactory
```

Comment:
The CaseFactory is not be used here. This can be removed. Although we could use it below